frontend/src/citizen-frontend/income-statements/IncomeStatements.tsx

```diff
@@ -44,7 +44,8 @@ function IncomeStatementsTable({
   const history = useHistory()
 
   const onOpen = useCallback(
-    (item: IncomeStatement) => () => history.push(`/income/${item.id}`),
+    (item: IncomeStatement, mode: 'view' | 'edit') => () =>
+      history.push(`/income/${item.id}/${mode}`),
```

Comment:
Answering myself from an earlier Q: I assume this is the reason for the view-suffix. I'd still prefer consistency over small code simplification (but I'd at least be fine with changing our style to _have_ the suffix; either way is fine IMO).